source/api_documentation/transactions.rst

```diff
@@ -187,6 +187,66 @@ The data returned has the following fields:
     - Promo Number ID
     - The Promo Number ID from the network.
 
+  * - signal_1_value
+    -
+    -
+
+  * - ...
+    -
+    -
+
+  * - signal_10_value
+    -
+    -
+
+  * - dynamic_number_pool_referrer_param11
+    -
+    -
+
+  * - dynamic_number_pool_referrer_param11_name
+    -
+    -
+
+  * - dynamic_number_pool_referrer_param11_value
+    -
+    -
+
+  * - ...
+    -
+    -
+
+  * - dynamic_number_pool_referrer_param25
+    -
+    -
+
+  * - dynamic_number_pool_referrer_param25_name
+    -
+    -
+
+  * - dynamic_number_pool_referrer_param25_value
```

Comment:
please move these DNPR param ones down just below the existing param1 ~ 10 section (or, may need to go below the `Potentially Extra Params` section, but I think better would be to keep it with other params but just call out in the description that these 11 ~ 25 are only available as an add-on feature.